### PR TITLE
Fix conflicts with upstream

### PR DESCRIPTION
**Pull request by voodoorai2000**
Objectives
==========
Fix conflicts with upstream

Notes
=====================
None

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -77,6 +77,7 @@ group :test do
   gem 'database_cleaner', '~> 1.6.1'
   gem 'email_spec', '~> 2.1.0'
   gem 'rspec-rails', '~> 3.6'
+  gem 'selenium-webdriver', '~> 3.10'
```
Comment (aitbw) [inline Gemfile]: This gem is on the custom Gemfile already, [here](https://github.com/AyuntamientoMadrid/consul/blob/master/Gemfile_custom#L4) —You can remove it from here or there, both are valid.

File: spec/features/budgets/votes_spec.rb [hunk 2]
```diff
@@ -45,7 +45,7 @@
         visit budget_investments_path(budget, heading_id: heading.id)
 
         within('.supports') do
-          find('.in-favor a').click
+          accept_confirm { find('.in-favor a').click }
```
Comment (aitbw) [inline spec/features/budgets/votes_spec.rb]: This is not needed here as it makes the test fail

File: spec/features/budgets/votes_spec.rb [hunk 3]
```diff
@@ -67,7 +67,7 @@
         visit budget_investment_path(budget, @investment)
 
         within('.supports') do
-          find('.in-favor a').click
+          accept_confirm { find('.in-favor a').click }
```
Comment (aitbw) [inline spec/features/budgets/votes_spec.rb]: This is not needed here as it makes the test fail

File: spec/features/budgets/votes_spec.rb [hunk 4]
```diff
@@ -78,7 +78,7 @@
...
```
Comment (aitbw) [inline spec/features/budgets/votes_spec.rb]: This is not needed here as it makes the test fail

File: spec/features/management/budget_investments_spec.rb [hunk 5]
```diff
@@ -215,14 +215,14 @@
       expect(page).to have_content(budget_investment.title)
 
       within("#budget-investments") do
-        find('.js-in-favor a').click
+        accept_confirm { find('.js-in-favor a').click }
```
Comment (aitbw) [inline spec/features/management/budget_investments_spec.rb]: This is not needed here as it makes the test fail

File: spec/features/budgets/votes_spec.rb (hunk 2 above)
Comment (bertocq) [reply spec/features/budgets/votes_spec.rb]: Why can't both codebases be equal? Is this going to also fail on other forks as well?

Comment (voodoorai2000) [reply spec/features/budgets/votes_spec.rb]: Yes... this is something pending to figure out...

The first time a user Supports an Investment, he sees a confirmation popup, which for some reason it wasn't required in tests...

We'll figure it out 😌

**Comment by voodoorai2000** [comment]
Appreciate the feedback @aitbw 👍
Fixed, let's see if Travis is happy now 😌

**Pull request merged**